### PR TITLE
Node hover

### PR DESCRIPTION
**Pull request by nerik**
Adds hover effect and cross-selection between sankey and map.

### CODE REVIEW

File: scripts/containers/titlebar.container.js [hunk 1]
```diff
@@ -7,7 +7,12 @@ const mapMethodsToState = (state) => ({
   selectNodes: state.flows.selectedNodesData,
   highlightNode: {
     _comparedValue: (state) => state.flows.highlightedNodeData,
-    _returnedValue: (state) => (state.flows.highlightedNodeData.length === 0) ? state.flows.selectedNodesData : state.flows.highlightedNodeData
+    _returnedValue: (state) => {
+      if (state.flows.highlightedNodeData.length === 0) {
```
Comment (agnlez) [inline scripts/containers/titlebar.container.js:11]: isn't this the same as `!state.flows.highlightedNodeData.length` ? ( 0 == falsy value)

File: scripts/components/titlebar.component.js [hunk 2]
```diff
@@ -4,21 +4,21 @@ export default class {
...
+    this._toggle(selectedNodesData.length > 0);
   }
 
-  _toggle(nodesData) {
-    if (nodesData.length === 0) {
-      this.search.classList.remove('is-hidden');
-      this.nodesTitles.classList.add('is-hidden');
-    } else {
+  _toggle(showTitles) {
+    if (showTitles === true) {
```
Comment (agnlez) [inline scripts/components/titlebar.component.js]: isn't `(showTitles)` enough?

File: scripts/components/titlebar.component.js [hunk 3]
```diff
@@ -3,13 +3,22 @@ export default class {
...
+    this._toggle(selectedNodesData.length > 0);
+  }
+
+  _toggle(showTitles) {
+    if (showTitles === true) {
       this.search.classList.add('is-hidden');
       this.nodesTitles.classList.remove('is-hidden');
+    } else {
+      this.search.classList.remove('is-hidden');
+      this.nodesTitles.classList.add('is-hidden');
```
Comment (agnlez) [inline scripts/components/titlebar.component.js]: I think you can use conditional `toggle` here, no?

**Comment by agnlez** [comment]
Should't the app updates when you click on a polygon in the map? Or this functionality is still not implemented?

**Comment by nerik** [comment]
Depends if the corresponding node exists in the left column of the sankey (most of the time it doesnt)
Yes, we need to explain visually that the node is disabled.

**Comment by agnlez** [comment]
Looks good!

File: scripts/components/titlebar.component.js (hunk 3 above)
Comment (nerik) [reply scripts/components/titlebar.component.js]: Yup

**Pull request merged**